src/Common/DTOs/Leaderboard/LeaderboardUserDto.cs

```diff
@@ -21,10 +21,8 @@ public class LeaderboardUserDto
 
     public LeaderboardUserDto() {}
 
-    public LeaderboardUserDto(User user, DateTime firstDayOfWeek)
+    public LeaderboardUserDto(User user)
```

Comment:
I note we still have a dependency on domain in the shared library here. We might be able to knock over #1093 and #538 in one go. @zacharykeeping please check with @AntPolkanov whether he is going to get around to this or if he is too busy with client work. Perhaps you or I can pick this one up. Or maybe @lukecookssw wants to do it.